docs/operational_guide/placement_configuration.md

```diff
@@ -154,3 +156,70 @@ curl -X POST <M3_COORDINATOR_HOST_NAME>:<M3_COORDINATOR_PORT(default 7201)>/api/
     ]
 }'
 ```
+
+#### Replacing a Seed Node
```

Comment:
Maybe add a disclaimer at the top of the file in bold saying something like: "If you're replacing an M3DB seed node please make sure to read the seed node specific instructions below."

Reply:
I mention this in `Replacing a Node`, we want it at the top as well?

Reply:
Yeah sorry I mean more like: "If you're performing any placement operations on an M3DB seed node..." Just a general disclaimer incase they miss this specific heading